dali/base/dadfs.cpp

```diff
@@ -10365,7 +10365,13 @@ IDFAttributesIterator *CDistributedFileDirectory::getDFAttributesIterator(const
     CMessageBuffer mb;
     mb.append((int)MDFS_ITERATE_FILES).append(wildname).append(recursive).append("").append(includesuper); // "" is legacy
     if (user)
-        user->serialize(mb);
+    {
+		Owned<IUserDescriptor> tmpUDesc = createUserDescriptor();
```

Comment:
odd indentation

Reply:
wouldn't it have been easier/cleaner to add a boolean parameter to user->serialize to indicate whether to include the pw token?